test/e2e/plain-v0_provisioner_test.go

```diff
@@ -345,6 +345,139 @@ var _ = Describe("plain-v0 provisioner bundleinstance", func() {
 		})
 	})
 
+	// TODO: try to create a BundleInstance that references a Bundle that contains
+	// existing resources.
+	When("a BundleInstance depends on another BundleInstance", func() {
+		var (
+			bundle *rukpakv1alpha1.Bundle
+			bi     *rukpakv1alpha1.BundleInstance
+			ctx    context.Context
+		)
+		BeforeEach(func() {
+			ctx = context.Background()
+
+			By("creating the testing OLM APIs Bundle resource")
+			bundle = &rukpakv1alpha1.Bundle{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "olm-apis",
+				},
+				Spec: rukpakv1alpha1.BundleSpec{
+					ProvisionerClassName: plainProvisionerID,
+					Image:                "quay.io/tflannag/olm-plain-bundle:olm-api-v0.20.0",
```

Comment:
Update now that we have locally built and loaded images?

Reply:
Definitely - I'm hoping to pick this back up sometime this week.

Reply:
I ended up creating a new PR as I didn't want to have to deal with a deleted file during rebasing this PR.